woocommerce/woocommerce
# [DataView]: Add Products View and Product Edit stories

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





This PR is a starting point for adding stories to the Product App project. We’re using Storybook as a playground to test different scenarios. We anticipate that these stories will evolve alongside the Product App itself.​

All the CSS changes are generated by [this script](https://github.com/woocommerce/woocommerce/blob/b0b9e65a3255ca052030bf273d8566dcb9e9c88d/tools/storybook/import-wp-css-storybook.sh), so it is not necessary review them.




### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Open the terminal in the `tools/storybook` path.
2. Run `pnpm storybook`.
3. Ensure that Storybook build correctly.
4. Ensure that Product Edit and Products view are displayed.


| Product Edit Story | Products View Story |
|--------|--------|
| ![image](https://github.com/user-attachments/assets/3482734f-194f-4ed3-84cb-3e0d2b3bc300) | ![image](https://github.com/user-attachments/assets/c56890fb-edf5-45b7-bc9b-e5e459331fc3)| 



### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

[DataView]: Add Products View and Product Edit stories.

</details>
